.travis.yml

```diff
@@ -22,6 +22,10 @@ php:
 
 matrix:
   allow_failures:
+    - php: 5.5
```

Comment:
Might as well remove these from the tests completely, right?

With this PR, the minimum version of PHP is basically 7.2, as everything lower doesn't get tested any more.
Shall we step up the move to PHP 7.2 as a minimum requirement for the core library?

Reply:
I think we should upgrade minimum PHP version to 7.2 for 1.x release only. Now lets use for development/testing only